sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -425,6 +515,14 @@ def _client_ready(self):
             self._link.attach()
             return False
         if self._link.get_state().value != 3:  # ATTACHED
+            if self._link.get_state().value == 6:
+                raise ErrorCondition.InternalError(
```

Comment:
What's this? ErrorCondition is an enum....
What's link state 6 and why would we be in it?

Reply:
Link state 6 is link error, I was trying to model after the uamqp client ready function. That is my bad, it should throw AMQPLinkError there if we want to raise an error I believe. Will follow up on this one